### DISCUSSION TITLE
Copy functionality for raw data in controls tab.

### DISCUSSION THREAD

**Discussion by sayinserdar** [storybookjs/storybook#15723] [Ideas]
Whenever I try to kickstart a project I use components from our storybook project but the default values of that component are too much for me to rewrite and duplicate. It would be fantastic to have a copy button for each raw data in the controls tab and a global copy button to copy all default values in the specified lang. Even exporting this raw data as a JSON file could be an option.

I'm willing to open up a PR if this is considered as  nice feature :v:

**Comment by shilman** [comment]
what exactly would it do?

**Reply by sayinserdar** [reply]
A shortcut to copy raw data to test it out your component with specific data in your own project (other than storybook).

**Reply by shilman** [reply]
Based on the current values of the controls?
